site/website/pages/en/demos.js

```diff
@@ -0,0 +1,187 @@
+const React = require('react');
+const CompLibrary = require('../../core/CompLibrary');
+const Container = CompLibrary.Container;
+const MarkdownBlock = CompLibrary.MarkdownBlock;
+
+class Demos extends React.Component {
+
+    render() {
+        const {config: siteConfig, language = ''} = this.props;
+        const cardStyle = {
```

Comment:
would it better to move these to the css?

Reply:
I moved these styles to a jss file.